bitnami/thanos/README.md

```diff
@@ -42,6 +42,24 @@ To uninstall/delete the `my-release` chart:
 helm uninstall my-release
 ```
 
+## Upgrading existing release to new major version
+
+### To 1.0.0
+If you are upgrading from a `<0.0.1` release you need to move your Querier Ingress information to the new values settings:
```

Comment:
Not only `0.0.1`, right? You need to implement those changes from any `0.X.X` release

Reply:
Yes by bad

Reply:
done